server/routes/helpers/validateIDCard.js

```diff
@@ -0,0 +1,43 @@
+
+/*
+* JavaScript implementation of the Luhn algorithm, with calculation and validation functions
+*/
+
+/* luhn_checksum
+ * Implement the Luhn algorithm to calculate the Luhn check digit.
+ * Return the check digit.
+ */
+function luhnChecksum(code) {
+  const len = code.length;
+  const parity = len % 2;
+  let sum = 0;
+  // eslint-disable-next-line no-plusplus
+  for (let i = len - 1; i >= 0; i--) {
+    // eslint-disable-next-line radix
+    let d = parseInt(code.charAt(i));
```

Comment:
You better specify the number 10 on parseInt since it will make sure that it parses it in base 10
```suggestion
    let d = parseInt(code.charAt(i), 10);
```